### PR TITLE
Feat/count of games by season kz

### PR DESCRIPTION
**Pull request by zkevkev**
This branch contains basic initialization of Game, League and Season. It also adds tests from the spec harness into the spec file which will contain a given method.

### CODE REVIEW

File: lib/stat_tracker.rb [hunk 1]
```diff
@@ -68,4 +66,13 @@ def game_data_creation(data)
     end
     game_data
   end
+
```
Comment (EricBelongea) [inline lib/stat_tracker.rb:69]: I'll assume that this works?

File: README.md
Comment (EricBelongea) [inline README.md:1]: perhaps we could do a \n so all of these could be a new line?

File: lib/game.rb
Comment (EricBelongea) [inline lib/game.rb:1]: I guess we'll find out, but do we need the other CSV data?

**Comment by billyburch** [comment]
I think this will be a supportive move to keep us organized moving forward. I appreciate the structure.

File: spec/game_spec.rb
Comment (EricBelongea) [inline spec/game_spec.rb:1]: Thank you for dividing all of the tests into logical spec files

**Review by EricBelongea — APPROVED** [review]
Left some comments, but I think this all looks good!

File: README.md
Comment (billyburch) [reply README.md:1]: I think the README is reader friendly as is, without the \n.  That said, I am wide open to the will of the group majority.

**Review by billyburch — APPROVED** [review]

File: lib/game.rb
Comment (zkevkev) [reply lib/game.rb:1]: I think most of the game methods are wrapped up in the game data, but we can always change this in future!

File: lib/game.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+class Game
+  def initialize(game_data)
```
Comment (vsanantone) [inline lib/game.rb:2]: I believe this line depending on the needs of the methods will be kept the same or changed as well for the other classes we built. Let's regroup near the mid point of our project to determine if we need to add arguments that pass in the CSV data into our initialize methods.

File: spec/game_spec.rb [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+  it "#percentage_visitor_wins" do
+    expect(@stat_tracker.percentage_visitor_wins).to eq 0.36
+  end
+
+  it "#percentage_ties" do
+    expect(@stat_tracker.percentage_ties).to eq 0.20
+  end
+
+  it "#count_of_games_by_season" do
+    expected = {
```
Comment (vsanantone) [inline spec/game_spec.rb:47]: Let's use the spec harness to write and modify our test. I like how the expected outcome gives us a format to work with and helps us better determine what we are looking for in the CSV data. We mentioned this in our discussions and let's use this for our advantage with the key => value data structure.

**Pull request merged**